components/navigation/navigation.js

```diff
@@ -1,6 +1,11 @@
-const navMenuIcon = document.querySelector("#nav-menu-icon");
-const navDropdownMenu = document.querySelector(".nav-dropdown-menu");
+const navMenuIcon = document.querySelector("#ham-icon");
+const navMenuCloseIcon = document.querySelector("#ham-close-icon");
+const navSideMenu = document.querySelector("#navigation-ham-menu");
 
 navMenuIcon.addEventListener("click", () => {
-navDropdownMenu.classList.toggle("active");
+    navSideMenu.classList.toggle("active");
+});
+
+navMenuCloseIcon.addEventListener("click", () => {
+    navSideMenu.classList.toggle("active");
 });
```

Comment:
You could add one empty line at the end of file to avoid such warnings by Github.

Reply:
Thanks for your suggestions, will work on them!